sqlmesh/core/engine_adapter/athena.py

```diff
@@ -317,6 +342,8 @@ def _query_table_type(
         Hit the DB to check if this is a Hive or an Iceberg table
         """
         table_name = exp.to_table(table_name)
+        quote_identifiers(table_name)
```

Comment:
why was this necessary?

Reply:
It's because the query its used in,  `SHOW TBLPROPERTIES {table}` is just a string and not an Expression.

since we are string building, we need to quote the identifiers manually. I couldn't figure out how to make it an Expression because sqlglot just turns it into an `exp.Command` string literal anyway

Reply:
ok, we can remove this and on line 349 do sql(dialect='hive', identify=True)

Reply:
Oh cool, forgot you could do that!